### PR TITLE
3 login and register pages

### LINKED ISSUE #3: Login and register pages
Only UI

### LINKED ISSUE #5: Theme mui
Stworzyć globalny theme i rozprowadzić go po aplikacji

Aktualnie ekran i logowanie są wzięte z [templatu](https://mui.com/material-ui/getting-started/templates/) (czyli tak się powinno robić, można się zainspirować jak używać tego MUI) i tak w nich to wygląda:

```tsx
import { createTheme, ThemeProvider } from '@mui/material/styles';

// TODO remove, this demo shouldn't need to reset the theme.
const defaultTheme = createTheme();

<ThemeProvider theme={defaultTheme}>
    // Zawartość strony
</ThemeProvider>
```

### PR DESCRIPTION
**Pull request by rubikon02**
image
image

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (Spookyless) [inline .idea/.gitignore]: Remove `/.idea` folder from being tracked

File: app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
+'use client'
```
Comment (Spookyless) [inline app/login/page.tsx:1]: Change language to English

File: app/register/page.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
+'use client'
```
Comment (Spookyless) [inline app/register/page.tsx:1]: Change language to English

**Review by Spookyless — CHANGES_REQUESTED** [review]

File: app/login/page.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        event.preventDefault();
+        const data = new FormData(event.currentTarget);
+        console.log({
+            email: data.get('email'),
+            password: data.get('password'),
+        });
+    };
+
+    return (
+        <ThemeProvider theme={defaultTheme}>
```
Comment (Spookyless) [inline app/login/page.tsx]: Move `ThemeProvider` to the application root, ie. one of the `layout.tsx` files

File: app/register/page.tsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            firstName: data.get('firstName'),
+            lastName: data.get('lastName'),
+            email: data.get('email'),
+            username: data.get('username'),
+            password: data.get('password'),
+        });
+    };
+
+    return (
+        <ThemeProvider theme={defaultTheme}>
```
Comment (Spookyless) [inline app/register/page.tsx]: Same as with `login/page.tsx`

File: .idea/.gitignore
Comment (ShatterPlayer) [inline .idea/.gitignore:1]: I am not sure whether editor configuration files should be part of the repository

File: .run/dev.run.xml
Comment (ShatterPlayer) [inline .run/dev.run.xml:1]: This file is also editor specific and should be rather local

Comment (rubikon02) [reply .run/dev.run.xml:1]: @Spookyless  should we keep .run in both repo? It's only for WebStorm and doesn't work in Visual Studio Code

Comment (Spookyless) [reply .run/dev.run.xml:1]: I'd say keep it

Comment (Spookyless) [reply .run/dev.run.xml:1]: We can keep it local yeah

**Review by Spookyless — APPROVED** [review]

**Pull request merged**